### ISSUE TITLE
Accessibility: IPython's homepage

### ISSUE THREAD

**Issue by Uzor13** [jupyterhub/outreachy#52]
### What page is this for?

https://ipython.org/

### WAVE accessibility report

<img width="360" alt="Screenshot 2022-10-09 at 1 22 23 AM" src="https://user-images.githubusercontent.com/40992425/195845153-3f05d87f-437a-4473-af4b-1f5c1ef89865.png">

- The document's language is not identified or a lang attribute value is invalid.
- Very low contrast between text and background colours. The current colour contrast is 4.07:1 instead of 4:5:1
- Fully justified text is present.
- Alternative text is likely insufficient or contains extraneous information.
- An image has a title attribute value but no alt value.
- Link text contains extraneous text or may not make sense out of context.

### The HTML element to be changed

```html
1. <html xmlns="http://www.w3.org/1999/xhtml">

2. HTML
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank">
Install
</a>

CSS
body {
  background-color: #fff;
}
a {
  color: #CE5C01;
}

3. HTML
<p>
To get started with IPython in the Jupyter Notebook, see our <a class="reference external" href="http://nbviewer.ipython.org/github/ipython/ipython/blob/master/examples/Index.ipynb">
official example collection
</a>
<a class="reference external" href="https://github.com/ipython/ipython/wiki/A-gallery-of-interesting-IPython-Notebooks">
notebook gallery
</a>
is an excellent way to see the many things you can do with IPython while learning about a variety of topics, from basic programming to advanced statistics or quantum mechanics.
</p> etc

CSS 
p {
  text-align: justify;
}

4. <img class="logo" src="_static/IPy_header.png" alt="Logo">

5. <img src="_static/nbviewer-thm-9.png" title="IPython Notebook Viewer">

6. on IPython’s Notebook Viewer
<a class="reference external" href="http://nbviewer.ipython.org/github/ptwobrussell/Mining-the-Social-Web-2nd-Edition/tree/master/ipynb/">
here
</a>
```

### Your proposed HTML (or CSS), after change

```html
1. <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
2. HTML
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank">
Install
</a>

CSS
body {
  background-color: #fff;
}
a {
  color: #bd5500; // increases the contrast to 4.7:1 which passes WCAG Guidelines.
}
3. 
CSS 
p {
  text-align: left;
} // makes it more readable without word breaks

4. <img class="logo" src="_static/IPy_header.png" alt="IPython Interactive Computing Logo">

5. <img src="_static/nbviewer-thm-9.png" alt="IPython Notebook Viewer">

6.<a class="reference external" href="http://nbviewer.ipython.org/github/ptwobrussell/Mining-the-Social-Web-2nd-Edition/tree/master/ipynb/">
on IPython’s Notebook Viewer
</a> // the name of the page rather than a generic here
```

### More info

Not in the wave report but semantic HTML can be used to improve and help screen readers easily know what part they are on
```{html}
<div class="header-wrapper">
<div class="header">
```
can be changed to:

```{html}
<header class="header">
<nav class="nav">
```
Another snippet:

```{html}
<div class="sphinxsidebar" role="navigation" aria-label="main navigation">
<div class="sphinxsidebarwrapper">
```
can be changed to:

```{html}
<section class="sphinxsidebar" aria-label=sidebar navigation">
<aside class="sphinxsidebarwrapper">
```
It makes the code more readable and accessible to tools and devices

**Comment by minrk** [comment]
Those are great suggestions! The semantic elements are definitely something we should consider proposing to the sphinx template folks.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
